maskrcnn_benchmark/engine/inference.py

```diff
@@ -6,16 +6,18 @@
 import os
 from collections import OrderedDict
 
-import torch
+import numpy as np
 
+from ..config import cfg
```

Comment:
I'd rather not import the config in this file, but keep it agnostic to it. You can probably pass the name of the dataset as an argument to the `inference` function

Reply:
Removed.